dipy/reconst/fwdti.py

```diff
@@ -0,0 +1,871 @@
+""" Classes and functions for fitting tensors without free water
+contamination """
+from __future__ import division, print_function, absolute_import
+
+import warnings
+
+import numpy as np
+
+import scipy.optimize as opt
+
+from dipy.reconst.base import ReconstModel
+
+from dipy.reconst.dti import (TensorFit, design_matrix, _min_positive_signal,
+                              decompose_tensor, from_lower_triangular,
+                              lower_triangular, mean_diffusivity)
+from dipy.reconst.dti import wls_fit_tensor as dti_wls_fit
+from dipy.reconst.dki import _positive_evals
+
+from dipy.reconst.vec_val_sum import vec_val_vect
+from dipy.core.ndindex import ndindex
+
+
+def fwdti_prediction(params, gtab, S0=1, Diso=3.0e-3):
+    """ Signal prediction given the free water DTI model parameters.
```

Comment:
If you are planning to have math in the docstring add r infront of the docstring. 
```
r"""
```